### PR TITLE
68 purging tests on angular10

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

medic/[angular10-migration#/68](https://github.com/medic/angular10-migration/issues/68)

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Comment by newtewt** [comment]
Overall nice changes getting these running again! Can you take a moment and move element definitions out of the specs and into a page object. Whether it's a new or existing page object.

**Comment by ngaruko** [comment]
> Overall nice changes getting these running again! Can you take a moment and move element definitions out of the specs and into a page object. Whether it's a new or existing page object.

Good point. Will wait though for https://github.com/medic/cht-core/pull/6777 to merge. I see a few page objects like `report list` that I was about to refactor

File: tests/e2e/login/purge.spec.js [hunk 1]
```diff
@@ -252,7 +253,8 @@ describe('Purging on login', () => {
     utils.resetBrowser();
     commonElements.goToLoginPage();
     loginPage.login(restrictedUserName, restrictedPass);
-    commonElements.calm();
+    //helper.waitUntilReady(element(by.css('#reports-list')));
```
Comment (newtewt) [inline tests/e2e/login/purge.spec.js:256]: any reason to keep the commented code?

**Review by newtewt — CHANGES_REQUESTED** [review]
Looks good, I would spend some time moving the element identifiers to a page object and remove commented code. Would also be good to migrate to async/await.

File: tests/page-objects/common/common.po.js [hunk 2]
```diff
@@ -158,5 +160,7 @@ module.exports = {
 
 function openSubmenu(menuName) {
   helper.findElementByTextAndClick(hamburgerMenuOptions, menuName);
-  helper.waitForAngularComplete();
+  //throwing error: `Cannot read property 'pendingRequests' of undefined` - 
+  //helper.waitForAngularComplete();
```
Comment (craig-landry) [inline tests/page-objects/common/common.po.js:164]: @ngaruko Did you mean to leave this in as a comment? Reading the comment it seems that the `helper.waitForAngularComplete()` call doesn't work, but I assume the fact that this PR is open means the tests run fine anyway. Does this commented-out code need to stay? I'd like to remove it if we can.

Also, I believe @newtewt had looked into the `waitForAngularComplete` stuff. @newtewt can you offer insight here?

Comment (ngaruko) [reply tests/page-objects/common/common.po.js:164]: Just left it for reference. We have had a discussion about it and concluded that it is not necessary. So I will remove that completely

**Pull request closed without merging**